mlflow/utils/autologging_utils.py

```diff
@@ -306,6 +310,36 @@ def batch_metrics_logger(run_id):
     batch_metrics_logger.flush()
 
 
+def _check_autologging_tested(module_name, lib_key, version_file_json):
+    min_version = version_file_json[lib_key]["autologging"]["minimum"]
+    max_version = version_file_json[lib_key]["autologging"]["maximum"]
+
+    pkg_ver = importlib.import_module(module_name).__version__
+
+    def get_minor_version(v):
+        return LooseVersion(v).version[:2]
+
+    return get_minor_version(min_version) <= get_minor_version(pkg_ver) \
+        <= get_minor_version(max_version)
+
+
+def is_autologging_tested(flavor_name):
+    version_file_path = resource_filename(__name__, "../ml-package-versions.yml")
+    with open(version_file_path) as f:
+        version_file_json = yaml.load(f, Loader=yaml.SafeLoader)
```

Comment:
can we load this file once at the module level to avoid reading / parsing YAML every time `autologging_is_disabled` is called?